apps/system/test/unit/app_usage_metrics_test.js

```diff
@@ -787,6 +812,7 @@ suite('AppUsageMetrics:', function() {
         assert.equal(keys[0], dayKey);
       });
 
+      assert.deepEqual(payload.searches, metrics.data.searches);
```

Comment:
can you add an assertion that your search for 'provider1' also exists in `payload.searches`?